docker-compose.yml

```diff
@@ -26,10 +26,11 @@ services:
       - .env
     volumes:
       - ./src:/app/src
-      - ./examples:/data
```

Comment:
I removed this because the file should have a more production-ready structure, and examples are copied to /data as explained in the readme